raven-log4j/src/main/java/com/getsentry/raven/log4j/SentryAppender.java

```diff
@@ -188,7 +196,9 @@ protected Event buildEvent(LoggingEvent loggingEvent) {
         }
 
         // Set culprit
-        if (loggingEvent.getLocationInformation().fullInfo != null) {
+        if (culpritPattern != null) {
+            eventBuilder.withCulprit(new PatternLayout("%c{3}").format(loggingEvent));
```

Comment:
`culpritPattern` isn't being used here, should it be in place of the literal `"%c{3}"`?